fcs/build.fsx

```diff
@@ -24,7 +24,7 @@ let isMono = false
 // Utilities
 // --------------------------------------------------------------------------------------
 
-let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.201"
+let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.300"
```

Comment:
@dsyme 
the DotnetCLIToolsVersion.txt has the version number of the dotnet cli to use.  Can you get the version by reading the version from this file?

https://github.com/Microsoft/visualfsharp/blob/master/DotnetCLIToolsVersion.txt

Reply:
thanks, now fixed